dbrequests/mysql/tests/test_mysql_connection.py

```diff
@@ -228,5 +223,6 @@ def is_none(x):
 
         db.send_data(df_update, 'membership', mode='truncate')
         df_in = db.send_query('SELECT * FROM membership')
-        assert is_none(df_in.membership[3])
-        assert is_none(df_in.average[3])
+        # The following is either a np.NaN (mysqldb) or None (pymysql)
+        assert np.isnan(df_in.membership[3]) or df_in.membership[3] is None
```

Comment:
In the is_none function it was working, but now it isnt if df_in.membership[3] is None, because np.isnan(None) is not working (great design)
```python
np.isnan(None) or None is None
Traceback (most recent call last):
  File "<input>", line 1, in <module>
TypeError: ufunc 'isnan' not supported for the input types, and the inputs could not be safely coerced to any supported types according to the casting rule ''safe''
```